tools/package_tor.js

```diff
@@ -0,0 +1,57 @@
+var execute = require('./lib/execute')
+const path = require('path')
+const fs = require('fs')
+
+var cmds = []
+
+const isWindows = process.platform === 'win32'
+const isDarwin = process.platform === 'darwin'
+const isLinux = process.platform === 'linux'
+
+const torS3Prefix = 'https://s3.us-east-2.amazonaws.com/demo-tor-binaries/'
+var torPath = process.argv.slice(2)[0] // npm run package-tor torPath
+if (torPath === undefined) {
+  torPath = path.join('app', 'extensions', 'bin')
+}
+
+var torVersion = '0.3.2.10'
+var braveVersion = '0.22.x'
```

Comment:
Suggestion: Just use a single number for this, e.g. 42 as in tor-3.2.10brave42, representing the number of the build of this tor executable.  I don't think it matters which version of Brave it is intended for (we might conceivably reuse it for several versions), but we may need to change it if we make a new build of tor with a different set of configuration options, for instance.

Reply:
@riastradh-brave  - can you try `npm install` in `browser-laptop` and confirm if the binary is present in `app/extensions/bin`.

Reply:
I tried

- `npm install`
- `rm -rf node_modules && npm install`
- `rm -rf brave-linux-x64 && rm -rf node_modules && npm install`

and none of them left a `tor` executable in app/extensions/bin, or a file named `tor` anywhere in the directory tree (except for `.git/.../refs/.../feature/tor` which is obviously something else!).

Reply:
pebcak